index.html

```diff
@@ -158,156 +161,70 @@ <h1 class="work-sample-2"> Multy-Post Stories </h1>
-				<a class="see" href="#"> See Project</a>
-			</div>
-			<div class="cards card-2">
-				<h1 class="work-sample"> Data Dashboard Health care </h1>
-				<p class="work-sample-text">
-					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-				</p>
-				<ul class="tags">
-					<li class="tag">html</li>
-					<li class="tag">bootstrap</li>
-					<li class="tag">ruby</li>
-				</ul>
-			</div>
-			<div class="cards card-3">
-				<br><br>
-				<h1 class="work-sample"> Website Protfolio </h1>
-				<p class="work-sample-text">
-					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-				</p>
-				<ul class="tags">
-					<li class="tag">html</li>
-					<li class="tag">bootstrap</li>
-					<li class="tag">ruby</li>
-				</ul>
+				<button class="see" type="button" onclick="myFunction(this)" id="3">See Projects</button>
 			</div>
+			<div class="cards card-2" id="cards-sample-1"></div>
+			<div class="cards card-3" id="cards-sample-2"></div>
 		</div>
 		<div class="row first-row">
-			<div class="cards card-1">
-				<h1 class="work-sample"> Profesional Art Printing Data </h1>
-				<p class="work-sample-text">
-					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-				</p>
-				<ul class="tags">
-					<li class="tag">html</li>
-					<li class="tag">bootstrap</li>
-					<li class="tag">ruby</li>
-				</ul>
-			</div>
-			<div class="cards card-2">
-				<h1 class="work-sample"> Data Dashboard Health care </h1>
-				<p class="work-sample-text">
-					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-				</p>
-				<ul class="tags">
-					<li class="tag">html</li>
-					<li class="tag">bootstrap</li>
-					<li class="tag">ruby</li>
-				</ul>
-			</div>
-			<div class="cards card-3">
-				<br><br>
-				<h1 class="work-sample"> Website Protfolio </h1>
-				<p class="work-sample-text">
-					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-				</p>
-				<ul class="tags">
-					<li class="tag">html</li>
-					<li class="tag">bootstrap</li>
-					<li class="tag">ruby</li>
-				</ul>
-			</div>
+			<div class="cards card-1" id="cards-sample-3"></div>
+			<div class="cards card-2" id="cards-sample-4"></div>
+			<div class="cards card-3" id="cards-sample-5"></div>
 		</div>
 	</div>
 	</div>
 </section>
+
+<section>
+<!-- Popups -->
+  <div id="myPopup" class="popup">
+
+  <!-- Modal content -->
+    <div class="popup-content">
+    	<span class="close">&times;</span>
+
+    	<div class="popup-flex-container">
+    		<div class="div-shadow-1" id="div-container">
+    		</div>
+    		<div class="div-shadow-2" id="div-container2">
+    		</div>
+    	</div>
+    	
+    </div>
```

Comment:
- [ ] Please follow the Figma design in your project popup window as you can see in the Figma design there is extra space at the bottom of the mobile popup window.

Your Design | Figma Design
---------------- | --------------------
![Screenshot from 2022-06-24 16-11-44](https://user-images.githubusercontent.com/33752735/175555342-8053c409-ef63-4f56-a608-5c4df9d8b943.png) | ![Screenshot from 2022-06-24 16-11-23](https://user-images.githubusercontent.com/33752735/175555372-b8c862fd-239f-442d-a07f-47f187169053.png)